templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -157,8 +157,11 @@
   {% block product_external_tab_combinations %}
     {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Combination/external_tab.html.twig') }}
   {% endblock %}
-{% endblock %}
 
+  {% block product_external_tab_category_tree %}
+    {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Category/external_category_tree_tab.html.twig') }}
```

Comment:
You don't need to call this one like an "external tab" because it's by default outside of the form since it is rendered inside a modal You could simply call it `category_tree_form.html.twig` for example

What I did with `Combination/external_tab` is a bit specific, it's rendered as if in the form (by cheating with the css so that the content is displayed like a regular tab) but it must not be in the product form itself (which contains all the tabs). Hence the "external tab" naming, which is not exactly what you are doing here.